.github/workflows/publish-dev-assets.yml

```diff
@@ -37,6 +37,10 @@ jobs:
           az version
           # Key Vault: 
           az account get-access-token --scope https://vault.azure.net/.default --output none
+      - name: Prepare notation certificate
+        run: |
+          mkdir -p truststore/x509/ca/ratify-verify
+          cp ./.well-known/pki-validation/ratify-verification.crt truststore/x509/ca/ratify-verify
```

Comment:
curious, do other project use difference certs for dev and prod assets?

Reply:
Good question. I can look around to see what's common.

Reply:
@susanshi @akashsinghal  Key-based signing and verification for Ratify is a short-term solution. We will shift to keyless solution in the near future. 

To reduce the maintenance cost of key/cert, I am thinking whether it is easier to use the same key/cert for both Ratify dev and prod assets?

Reply:
I maybe ask these question a bit late or miss something: why do we sign dev images?

Reply:
We started with dev image signing initially so that we can establish our image signing guidance and see if we have any feedback from community. We haven't heard anything so we are proceeding with release image signing. There's no requirement to have dev image signing but since we already do, I'm thinking we can just keep it.

Reply:
Spoke with @yizha1 offline and he's fine with signing dev images as well.